Description: Remove LimitNOFILE from docker.service
Author: Tianon <tianon@debian.org> (& Brennan Kinney)
Applied-Upstream: 25+
Forwarded: https://github.com/moby/moby/pull/45534

See also https://github.com/containerd/containerd/pull/8924

diff --git a/contrib/init/systemd/docker.service b/contrib/init/systemd/docker.service
index 8275401b1a..d8c7867057 100644
--- a/contrib/init/systemd/docker.service
+++ b/contrib/init/systemd/docker.service
@@ -28,7 +28,6 @@ StartLimitInterval=60s
 
 # Having non-zero Limit*s causes performance problems due to accounting overhead
 # in the kernel. We recommend using cgroups to do container-local accounting.
-LimitNOFILE=infinity
 LimitNPROC=infinity
 LimitCORE=infinity

